taskcluster/kinds/alignments-backtranslated/kind.yml

```diff
@@ -79,6 +79,7 @@ tasks:
                     --corpus_src=$MOZ_FETCHES_DIR/mono.{src_locale}.zst
                     --corpus_trg=$MOZ_FETCHES_DIR/mono.{trg_locale}.zst
                     --output_path=$TASK_WORKDIR/artifacts/mono.aln.zst
+                    --output_tokenized
                     --tokenization=moses
```

Comment:
Thought: So thinking through the state of the pipeline, every language will go through tokenized, and it will use Moses. Moses is not great at CJK, but it works well for many other languages. So eventually we'll need better tokenization (one option being #860). But the implementation should be correct for _SOME_ kind of tokenization happening that we can swap in different tokenizers as we decide what's best for everything, or even per-lang tokenizers if we decide to go that route.

Reply:
yes, the tokenization method is an abstraction from alignments and pipeline perspective. The only limitation now is compatibility with OpusTrainer for detokenization